### DISCUSSION TITLE
Search fails when getting results for one input then adding characters to query

### DISCUSSION THREAD

**Discussion by mendiromania** [requarks/wiki#3701] [Error / Bug Report]
**Describe the bug**//**To Reproduce**
https://docs.requarks.io/ in search bar type "in", wait for results, then type "stall" to make the word "install"

**Expected behavior**
Pages matching "install" are returned

**Screenshots**
https://i.imgur.com/Bw2KrM0.png

**Host Info (please complete the following information):**
https://docs.requarks.io/

**Comment by ravenzachary** [comment]
I am also seeing this same bug manifest when you search for one thing, then delete the search field and enter something entirely different. The only workaround at this time is to reload the page and search again.
